circuitpython_kernel/kernel.py

```diff
@@ -66,13 +67,13 @@ def run_code(self, code):
             result.extend(self.serial.read_all())
 
         assert result.startswith(b'OK')
-        out, err = result[2:-2].split(b'\x04', 1) # split result
+        out, err = result[2:-2].split(b'\x04', 1)  # split result
 
         return out.decode('utf-8', 'replace'), err.decode('utf-8', 'replace')
 
-
-    def do_execute(self, code, silent, store_history=True,
-                   user_expressions=None, allow_stdin=False):
+    def do_execute(
+        self, code, silent, store_history=True, user_expressions=None, allow_stdin=False
```

Comment:
Pylint is tossing a bad-continuation error here (C: 75, 0: Wrong hanging indentation before block (add 4 spaces)). Could you fix this and push to the _style-py_ branch again? Thanks!